packages/orchestration/src/orchestration-types.ts

```diff
@@ -446,13 +445,52 @@ export interface TranslationConfigParams {
   targetLanguage: string;
 }
 
+/**
+ * Output parameters for translation output configuration.
+ */
+export interface TranslationOutputParameters {
+  /**
+   * Language of the text to be translated.
+   * @example sourceLanguage: "de-DE"
```

Comment:
```suggestion
   * @example sourceLanguage: 'de-DE'
```

Should not suggest using `"`